pym/gentoolkit/eclean/cli.py

```diff
@@ -590,9 +602,13 @@ def doAction(action, options, exclude={}, output=None):
             output.einfo("Cleaning " + files_type + "...")
         # do the cleanup, and get size of deleted files
         if options["pretend"]:
-            clean_size = cleaner.pretend_clean(clean_me)
+            if options["skip-git"]:
+                git = {}
+            clean_size = cleaner.pretend_clean(clean_me, git)
         elif action in ["distfiles"]:
-            clean_size = cleaner.clean_dist(clean_me)
+            if options["skip-git"]:
```

Comment:
I think we should call the option `skip-vcs` in case we add support for other VCSes (in the respective eclasses) in future?

Reply:
Yes sam ... done